src/Analysis/GWASV2/Components/SelectCohort/SelectCohortDropDown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery, queryConfig } from 'react-query';
+import { Select, Spin } from 'antd';
+import { fetchCohortDefinitions } from '../../Shared/cohortMiddlewareApi';
+import { useFetch } from '../../Shared/formHooks';
+import { useSourceContext } from '../../Shared/Source';
+
+const SelectCohortDropDown = ({
+  selectedCohort = undefined,
```

Comment:
I think we can omit the default prop declaration here since it is defined at 62. Consider instantiating a default value with ```null``` instead of undefined, to be consistent with rest of this codebase.

Reply:
this prop has been removed altogether.